YogaKit/YGLayout.m

```diff
@@ -318,6 +312,14 @@ static void YGAttachNodesFromViewHierachy(UIView *const view)
   if (yoga.isLeaf) {
     YGRemoveAllChildren(node);
     YGNodeSetMeasureFunc(node, YGMeasureView);
+
+    YGSize oldSize = yoga.cachedSize;
+    YGSize newSize = YGMeasureView(node, 0, YGMeasureModeUndefined, 0, YGMeasureModeUndefined);
```

Comment:
@emilsjolander @dshahidehpour Point taken. But in that case, I'd still argue that this feature is worthwhile for quite a few projects where the performance impact won't be felt. I use it myself on a project and I'm still at 60FPS. Could we introduce the feature as an **off-by-default** feature as suggested?